app/presenters/local_authority_external_content_presenter.rb

```diff
@@ -0,0 +1,19 @@
+class LocalAuthorityExternalContentPresenter
```

Comment:
Hmm. So, both of these classes are 100% covered by the model callback tests, and I feel like that's okay for the publisher class because all that's doing is abstracting the api calls, so probably unit test doesn't add anything. 

However, thinking this through to defend it, I think the Presenter class probably does want a unit test, because it's used as both test value creation and code path in the feature test, so if it breaks it won't necessarily fail that test. I'll add a unit test for it.

Reply:
Done!